docs/en/concepts-and-designs/overview.md

```diff
@@ -1,45 +1,30 @@
 # Overview
-SkyWalking: an open source observability platform to collect, analysis, aggregate and visualize data from services and cloud native 
-infrastructures. SkyWalking provide an easy way to keep you have clear view about your distributed system, even across Cloud.
+SkyWalking: an open source observability platform to collect, analyze, aggregate and visualize data from services and cloud native infrastructures. SkyWalking provide an easy way to keep you have a clear view of your distributed system, even across Cloud.
 It is more like an modern APM, especially designed for cloud native, container based and distributed system.
 
 ## Why use SkyWalking?
-SkyWalking provides solutions for observing and monitoring distributed system, in many different scenarios. First of all,
-like traditional ways, SkyWalking provides auto instrument agents for service, such as Java, C#
-and NodeJS. At same time, provide manual instrument SDKs for Go(Not yet), C++(Not yet). 
-Also with more languages required, risks in manipulating codes at runtime, cloud native infrastructures grow 
-more powerful, SkyWalking could use Service Mesher infra probes to collect data for understanding the whole distributed system.
-In generally, it provides observability capabilities for **service**(s), **service instance**(s), **endpoint**(s).
+SkyWalking provides solutions for observing and monitoring distributed system, in many different scenarios. First of all, like traditional ways, SkyWalking provides auto instrument agents for service, such as Java, C# and NodeJS. At the same time, it provides manual instrument SDKs for Go(Not yet), C++(Not yet). Also with more languages required, risks in manipulating codes at runtime, cloud native infrastructures grow powerful, SkyWalking could use Service Mesher infra probes to collect data for understanding the whole distributed system. In generally, it provides observability capabilities for **service**(s), **service instance**(s), **endpoint**(s).
 
-Service, instance and endpoint concepts are using everywhere in today, so let's be clear what we mean these in SkyWalking first.
+The concept of Service, Instance and Endpoint are used everywhere today, so let's be clear about what they mean in SkyWalking first.
```

Comment:
The concepts.

Reply:
I'll check the format, it should be caused by the markdown editor.